modules/snooker/src/main/ts/ephox/snooker/api/TableOperations.ts

```diff
@@ -305,8 +321,8 @@ export const splitCellIntoColumns = run(opSplitCellIntoColumns, onCell, resize,
 export const splitCellIntoRows = run(opSplitCellIntoRows, onCell, Fun.noop, Fun.noop, Generators.modification);
 export const eraseColumns = run(opEraseColumns, onCells, resize, prune, Generators.modification);
 export const eraseRows = run(opEraseRows, onCells, Fun.noop, prune, Generators.modification);
-export const makeColumnHeader = run(opMakeColumnHeader, onCell, Fun.noop, Fun.noop, Generators.transform('row', 'th'));
-export const unmakeColumnHeader = run(opUnmakeColumnHeader, onCell, Fun.noop, Fun.noop, Generators.transform(null, 'td'));
+export const makeColumnHeader = run(opMakeColumnHeader, onCells, Fun.noop, Fun.noop, Generators.transform('row', 'th'));
+export const unmakeColumnHeader = run(opUnmakeColumnHeader, onCells, Fun.noop, Fun.noop, Generators.transform(null, 'td'));
 export const makeRowHeader = run(opMakeRowHeader, onCell, Fun.noop, Fun.noop, Generators.transform('col', 'th'));
```

Comment:
Was there a reason this also wasn't changed, as it appears it has the same issue? If we're changing one, we should do both so as to keep things consistent.

Reply:
What is 'this' in this situation? `makeRowHeader`?

Reply:
Yes, that's the line I've commented on. Sorry it wasn't clear.

Reply:
There's also `unmakeRowHeader` below that this also applies to sorry.

Reply:
Wasn't really mentioned in the task, so I wasn't sure if I should change it or not.
But I'll make the change.